### PR TITLE
[zh-cn]Update cluster-upgrade.md

### PR DESCRIPTION
**Pull request by yanrongshi**
Preview:https://github.com/kubernetes/website/pull/35071/files?short_path=7b82cdf#diff-7b82cdf19706f810f71fa7b368b797e17bd50f8d93bda5a17cbf85b3754daa00

### CODE REVIEW

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 1]
```diff
@@ -112,8 +112,8 @@ kubelet on that node and bring the node back into service.
 [安装最新版本的 `kubectl`](/zh-cn/docs/tasks/tools/).
 
 对于集群中的每个节点，
-[排空](/zh-cn/docs/tasks/administer-cluster/safely-drain-node/)
-节点，然后，或者用一个运行了 {{< skew currentVersion >}} kubelet 的新节点替换它；
+首先需要[排空](/zh-cn/docs/tasks/administer-cluster/safely-drain-node/)
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:115]:
```suggestion
首先需要[腾空](/zh-cn/docs/tasks/administer-cluster/safely-drain-node/)
```

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 2]
```diff
@@ -112,8 +112,8 @@ kubelet on that node and bring the node back into service.
...
+节点，然后使用一个运行了 kubelet {{< skew currentVersion >}} 版本的的新节点替换它；
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:116]:
```suggestion
节点，然后使用一个运行了 kubelet {{< skew currentVersion >}} 版本的新节点替换它；
```

File: content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md [hunk 3]
```diff
@@ -152,7 +152,7 @@ write it back also using the latest supported API.
 当底层的 API 更改时，这些对象可能需要用新 API 重写。
 如果不能做到这一点，会导致再也不能用 Kubernetes API 服务器解码、使用该对象。
 
-对于每个受影响的对象，用最新支持的 API 获取它，然后再用最新支持的 API 写回来。
+对于每个受影响的对象，请使用最新支持的 API 获取它，然后也使用最新支持的 API 将其写回。
```
Comment (tengqm) [inline content/zh-cn/docs/tasks/administer-cluster/cluster-upgrade.md:155]:
```suggestion
对于每个受影响的对象，请使用最新支持的 API 读取它，然后使用所支持的最新 API 将其写回。
```

**Comment by yanrongshi** [comment]
@tengqm cc

**Pull request merged**